packages/hint-strict-transport-security/src/hint.ts

```diff
@@ -160,7 +160,11 @@ export default class StrictTransportSecurityHint implements IHint {
             if (!isHTTPS(resource) && headerValue) {
                 const message = `'strict-transport-security' header should't be specified in pages served over HTTP.`;
 
-                context.report(resource, message, { element });
+                context.report(resource, message, {
+                    codeLanguage: 'http',
+                    codeSnippet: `${capitalizeHeaderName('strict-transport-security')}: ${headerValue}`,
```

Comment:
Since we're hard-coding the header name anyway using `capitalizeHeaderName` seems a bit overkill...
```suggestion
                    codeSnippet: `Strict-Transport-Security: ${headerValue}`,
```

Reply:
Completely agree.